tests/subsys/fs/fcb/testcase.yaml

```diff
@@ -12,6 +12,20 @@ tests:
     tags: flash_circural_buffer
     integration_platforms:
       - nrf52840dk/nrf52840
+  filesystem.fcb.fixed_endmarker:
+    platform_allow:
+      - nrf52840dk/nrf52840
+      - nrf52dk/nrf52832
+      - nrf51dk/nrf51822
+      - native_posix
+      - native_posix/native/64
```

Comment:
why  native_posix? this should already be covered with native_sim. I would remove this from other test scenarios as well.

Reply:
Removed the native_posix targets